packages/front-end/components/Radix/Callout.tsx

```diff
@@ -0,0 +1,25 @@
+import { Callout as RadixCallout } from "@radix-ui/themes";
+import { ReactNode } from "react";
+import { RadixStatusIcon, Status, getRadixColor } from "./HelperText";
+import styles from "./RadixOverrides.module.scss";
+
+export default function Callout({
+  children,
+  type,
+}: {
+  children: ReactNode;
+  type: Status;
```

Comment:
Lets use the same `status` prop name as we do with HelperText to keep things consistent.

Reply:
Also, follow the pattern in Checkbox where you merge in `MarginProps` to the accepted props and pass them to the outer container.  This will allow getting rid of wrapping `<div className="mt-3">`... components and specifying margins directly.